tokio/src/io/util/copy.rs

```diff
@@ -23,7 +23,7 @@ impl CopyBuffer {
             pos: 0,
             cap: 0,
             amt: 0,
-            buf: vec![0; 2048].into_boxed_slice(),
+            buf: vec![0; 8192].into_boxed_slice(),
```

Comment:
Alternatively, we can use the `DEFAULT_BUF_SIZE` constant.
https://github.com/tokio-rs/tokio/blob/09b770c5db31a1f35631600e1d239679354da2dd/tokio/src/io/util/mod.rs#L87

```suggestion
            buf: vec![0; super::DEFAULT_BUF_SIZE].into_boxed_slice(),
```